infractions.yml

```diff
@@ -1,36 +1,48 @@
 ---
+# Positive associations with "white" (like whitelisted) and negative associations with "black"
+# (like blacklisted) recall a long legacy of violence and oppression against Black and
+# African-American populations. Prefer terms that do not ascribe positive or negative meaning to
+# colors.
 - terms:
   - whitelist
   - white-list
   alternatives:
   - allowlist
-
 - terms:
   - blacklist
   - black-list
   alternatives:
   - denylist
 
+# Grandfather clauses were included in discriminatory laws in the United States. The laws made it
+# more difficult for African Americans to register to vote. Prefer describing exemptions and
+# exceptions without recalling this history.
 - terms:
   - grandfather
   alternatives:
   - exempted
 
+# This term has been used as a slur to refer to individuals with hearing disabilities. Prefer more
+# descriptive alternatives for describing an object that is standing in for something else.
 - terms:
   - dummy
   alternatives:
   - placeholder
   - fake
   - stub
 
-# This term can be appropriate when used in the context of stress testing.
+# The term "redlining" is rooted in discriminatory practices in the United States of denying
+# services like mortgages or insurance based on geographical locations. Describe annotations on
```

Comment:
nit: Maybe "Prefer describing annotations..." so that all of the final sentences in these blurbs have the same structure?

Reply:
👍  f17cab7